src/licensedcode/data/rules/gpl-3.0-plus_22.yml

```diff
@@ -0,0 +1,3 @@
+licenses:
+    - gpl-2.0
+notes: no version is specified and that the original archive has both a GPL 2.0 and 3.0 texts

```

Comment:
@pombredanne  I'm not sure about the format for the `notes` parameter in a `.yml` file. can I write like this : 
```
licenses:
    - gpl-2.0
notes: first line
second line
third line
.
.
```